v56-team22-surgery-status-board/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,126 @@
+const Footer = () => {
+  return (
```

Comment:
I think the footer is looking good and there's high contrast between the green we're using and the darker text which is great for accessibility! 

How would you feel about making the code a little more dry and modular by abstracting the developer names into a `developers` array and the styling into a `DeveloperItem` that takes `name` as a prop? 

You can then loop through the `developers` array in the Footer component and return a `DeveloperItem` with each person's name. This will make the Footer code a bit more readable.